### PR TITLE
Fix NameError

### LINKED ISSUE #1: Fix undefined error
See https://github.com/HenryJFlynn/mouse2afc/issues/4#issuecomment-1552667899 and https://github.com/HenryJFlynn/mouse2afc/issues/4#issuecomment-1552739317.

### PR DESCRIPTION
**Pull request by HenryJFlynn**
Closes https://github.com/HenryJFlynn/pybpod-api/issues/1.

### CODE REVIEW

File: pybpodapi/bpod/emulator/emulator.py [hunk 1]
```diff
@@ -76,6 +85,7 @@ def __init__(self, hardware):
         self._state = State(self.hardware)
         self._manual_override_events = queue.Queue()
         self._socket = None
+        self.emulator_mode = GUI_ENABLED
```
Comment (ckaraneen) [inline pybpodapi/bpod/emulator/emulator.py]: I don't understand this. Why define an emulator mode attribute in the emulator class?

File: pybpodapi/bpod/emulator/emulator.py [hunk 2]
```diff
@@ -220,40 +230,32 @@ def _set_global_timer_channel(self, channel, on):
...
+            output_channel_name = \
+                self._hardware.channels.output_channel_names[
+                    channel_number]
+            message = f'{output_channel_name}:{value}'
+            self.socket.write(message.encode('utf-8'))
+            if not self.socket.waitForBytesWritten(2000):
                 error_message = \
-                    f'Could not connect to server: {self.socket.errorString()}'
+                    'Could not write to socket: ' + \
+                    self.socket.errorString()
```
Comment (ckaraneen) [inline pybpodapi/bpod/emulator/emulator.py]: No need to modify this string.

**Review by ckaraneen — CHANGES_REQUESTED** [review]

File: pybpodapi/bpod/emulator/emulator.py (hunk 1 above)
Comment (HenryJFlynn) [reply pybpodapi/bpod/emulator/emulator.py]: Fixed.

File: pybpodapi/bpod/emulator/emulator.py (hunk 2 above)
Comment (HenryJFlynn) [reply pybpodapi/bpod/emulator/emulator.py]: Fixed.

**Review by ckaraneen — APPROVED** [review]
Looks fine.

**Pull request merged**